src/resmom/mom_comm.c

```diff
@@ -5439,6 +5439,8 @@ im_request(int stream, int version)
 				 	 	"strdup failure setting nodehost");
 					goto err;
 				}
+				clear_attr(&pjob->ji_resources[resc_idx].nr_used,
+						&job_attr_def[JOB_ATR_resc_used]);
```

Comment:
Do you need to check nr_used.flags for ATR_VFLAG_SET? Is there the potential to leak memory here?

Reply:
No because this is the part in the code where we’re initializing/filling in data for a new slot: pjob->ji_resources[resc_idx].
It has not been set yet.